File: app/src/main/java/com/mundi/databaseadmin/database/CreateDatabase.java
package com.mundi.databaseadmin.database;

import android.util.Log;

import java.util.ArrayList;

public class CreateDatabase {

    private static final String TAG = "CreateDatabase";

    private BackendConnection backendConnection;
    public CreateDatabase(String uri, String databasename) {
        Log.d(TAG, "ListTables()");
        String cmd = "createdatabase&dbname=" + databasename;
        backendConnection = new BackendConnection(uri, cmd);
        return;
    }

    public boolean getResponse() {
        Log.d(TAG, "getResponse()");
        if (backendConnection.getResposeCode() == 200) {
            return true;
        }
        return false;
    }

    public String getResponseString() {
        Log.d(TAG, "getResponseString()");
        return backendConnection.getResponseString();
    }

}

File: app-kotlin/src/main/java/com/mundi/databaseadmin/database/CreateDatabase.kt
package com.mundi.databaseadmin.database

import android.util.Log

class CreateDatabase(uri: String?, databasename: String) {
    private val backendConnection: BackendConnection
    val response: Boolean
        get() {
            Log.d(TAG, "getResponse()")
            return if (backendConnection.resposeCode == 200) {
                true
            } else false
        }

    val responseString: String
        get() {
            Log.d(TAG, "getResponseString()")
            return backendConnection.responseString
        }

    companion object {
        private const val TAG = "CreateDatabase"
    }

    init {
        Log.d(TAG, "ListTables()")
        val cmd = "createdatabase&dbname=$databasename"
        backendConnection = BackendConnection(uri, cmd)
        return
    }
}